src/main.rs

```diff
@@ -178,3 +181,93 @@ pub async fn main() -> Result<(), anyhow::Error> {
     }
     Ok(())
 }
+
+fn parse_env_var(val: &str) -> anyhow::Result<(String, String)> {
+    let (key, value) = val
+        .split_once('=')
+        .ok_or_else(|| anyhow::anyhow!("Invalid environment variable, did not find '='"))?;
+
+    // Check if the key is empty (i.e. the user set -e =bar). An environment variable must have a
+    // key, but can have an empty value
+    if key.is_empty() {
+        return Err(anyhow::anyhow!(
+            "Environment variable must have a non-empty key"
+        ));
+    }
+
+    // If the value starts and ends with a double or single quote, assume it is a quoted value and
+    // strip the quotes
+    let final_value = if value.starts_with('"') && value.ends_with('"') {
+        value.trim_matches('"').to_owned()
+    } else if value.starts_with('\'') && value.ends_with('\'') {
```

Comment:
Glad you included single quotes. That actually makes more sense usability-wise than double quotes.